src/step.rs

```diff
@@ -28,6 +28,7 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
     /// Returns true as long as there are more things to do.
     pub fn step(&mut self) -> EvalResult<'tcx, bool> {
+        self.memory.clear_packed();
```

Comment:
Either here or on the definition of `clear_packed` it'd be nice to have a comment saying "see the docs on the `packed` field for why we do this".